chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java

```diff
@@ -401,6 +401,9 @@ public DeveloperDTO merge(List<Long> developerIdsToMerge, DeveloperDTO developer
 
     @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).DEVELOPER, "
             + "T(gov.healthit.chpl.permissions.domains.DeveloperDomainPermissions).SPLIT, #oldDeveloper)")
+    @CacheEvict(value = {
+            CacheNames.DEVELOPER_NAMES
+    }, allEntries = true)
```

Comment:
Does that job evict the caches using the @CacheEvict annotation? That is the only mechanism to send an eviction message to the other tomcat node.

Reply:
If you look at SplitDeveloperJob there is this line of code:                ` CacheManager.getInstance().clearAll();`
 . I thought we did it that way because putting CacheEvict on the execution method of a quartz job didn't work but I don't recall for sure. Are you saying the cache clearing code in the job won't work?

Reply:
Hmm. That clearAll() code should work, but it does not send a message to the other nodes to clear their cache - the annotations are the mechanism that send messages. Might be something that needs to be tested, maybe?